addon/components/nrg-checkbox.hbs

```diff
@@ -0,0 +1,20 @@
+<div
+  class="ui checkbox
+    {{if @disabled "disabled"}}
+    {{if @fitted "fitted"}}
+    {{if @readonly "read-only"}}
+    {{@class}}"
+>
+  <Input
```

Comment:
I noticed for `<NrgTextField::TrimInput />` and #324 you used the basic HTML `<input />` and `<textarea />` tags, but here you're using Ember's `<Input />` component - what was the reason for that?

Reply:
If you use the Ember component `<Input />` I expect you'll need to prefix the properties with an `@`, eg.
```js
  @id={{this.focusId}}
  @type="checkbox"
  @checked={{this.checked}}
```

Reply:
Good catch. This was not intended. I'm trying to use raw html tags wherever possible.